themes/src/main/resources-community/theme/base/admin/messages/messages_ka.properties

```diff
@@ -15,3 +15,51 @@ invalidPasswordMinSpecialCharsMessage=არასწორი პაროლ
 invalidPasswordNotUsernameMessage=არასწორი პაროლი: არ უნდა იყოს მომხმარებლის სახელის ტოლი.
 invalidPasswordNotEmailMessage=არასწორი პაროლი: არ უნდა უდრიდეს ელფოსტას.
 invalidPasswordRegexPatternMessage=არასწორი პაროლი: არ ემთხვევა რეგგამოსის ნიმუშ(ებ)-ს.
+ldapErrorMissingClientId=კლიენტის ID-ის მითითება აუცილებელია კონფიგურაციაში, როცა რეალმის როლების ასახვა არ გამოიყენება.
+ldapErrorCantPreserveGroupInheritanceWithUIDMembershipType=ჯგუფის მემკვიდრეობითობის შენარჩუნება და UID წევრობის ტიპს ერთად ვერ გამოიყენებთ.
+ldapErrorCantWriteOnlyForReadOnlyLdap=მხოლოდ, ჩაწერის დაყენება შეუძლებელია, როცა LDAP მომწოდებლის რეჟიმი WRITABLE არაა
+ldapErrorCantEnableStartTlsAndConnectionPooling=ორივეს, StartTLS და მიერთების პულინგს ერთდროულად ვერ გამოიყენებთ.
+ldapErrorCantEnableUnsyncedAndImportOff=მომხმარებლის შემოტანის გათიშვა შეუძლებელია, როცა LDAP მომწოდებლის ტიპი წარმოადგენს: UNSYNCED
+ldapErrorValidatePasswordPolicyAvailableForWritableOnly=პაროლების პოლიტიკის გადამოწმება, მხოლოდ, WRITABLE ჩასწორების რეჟიმში შეიძლება
+clientRedirectURIsFragmentError=გადასამისამართებელი URI-ები URI ფრაგმენტს არ უნდა შეიცავდეს
+clientRootURLIllegalSchemeError=ძირითადი URL არასწორ სქემას იყენებს
+backchannelLogoutUrlIllegalSchemeError=Backchannel-ის გასვლის URL არასწორ სქემას იყენებს
+clientRedirectURIsIllegalSchemeError=გადამისამართების URI არასწორ სქემას იყენებს
+clientBaseURLInvalid=საბაზისო URL სწორი URL არაა
+clientRootURLInvalid=საწყისი URL სწორი URL არაა
+clientRedirectURIsInvalid=გადამისამართების URI სწორი URI არაა
+backchannelLogoutUrlIsInvalid=უკანა არხის გასვლის URL სწორი URL არაა
+
+
+pairwiseMalformedClientRedirectURI=კლიენტი არასწორ გადამისამართების URI-ს შეიცავდა.
+pairwiseClientRedirectURIsMissingHost=კლიენტის გადამისამართების URI-ები სწორ ჰოსტის კომპონენტს უნდა შეიცავდეს.
+pairwiseRedirectURIsMismatch=კლიენტის გადამისამართების URI-ები არ ემთხვევა გადამისამართების URI-ებს, რომელიც სექტორის იდენტიფიკატორის URI-დან მივიღე.
+error-invalid-value=არასწორი მნიშვნელობა.
+error-invalid-blank=შეიყვანეთ მნიშვნელობა.
+error-empty=შეიყვანეთ მნიშვნელობა.
+error-invalid-length-too-short=ატრიბუტის {0} მინიმალური სიგრძეა {1}.
+error-invalid-email=არასწორი ელფოსტის მისამართი.
+error-number-out-of-range-too-big=ატრიბუტის {0} მაქსიმალური მნიშვნელობაა {2}.
+error-pattern-no-match=არასწორი მნიშვნელობა.
+error-person-name-invalid-character={0} არასწორ სიმბოლოს შეიცავს.
+ldapErrorCantWriteOnlyAndReadOnly=მხოლოდ-ჩაწერად და მხოლოდ-წაკითხვად ალმებს ერთდროულად ვერ დააყენებთ
```

Comment:
Could you please check this translation? When using Google translate, it translates to "You cannot set write-only and read-only alms at the same time" where "alms" should probably be "attribute". Maybe it is just an error in Google translate, still I'd like you to double-check. Thanks!

Reply:
Well, ალმები - almebi - flags - can be said anything which is either set or not. I can translate it as attribute too, np to me if you prefer so.

Reply:
Just see it as a comment from me, I'll leave that to a native speaker to decide. I see that the English original is a bit vague as well, as it states "Can not set write-only and read-only together", as it should probably read "Can not set write-only and read-only for an attribute". Still, changing the English original is out of scope for this PR.